eip-x.md

```diff
@@ -51,4 +51,4 @@ Test cases for an implementation are mandatory for EIPs that are affecting conse
 The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
 
 ## Copyright
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Please don't change or rename this file.

Reply:
I thought I reverted.

Reply:
Fixed it for you.